manifests/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -287,6 +241,40 @@ data:
 {{ toYaml .Values.global.proxy.envoyAccessLogService.tcpKeepalive | trim | indent 10 }}
       {{- end}}
     {{- end}}
+{{- end }}
+
+{{/* We take the mesh config above, defined with individual values.yaml, and merge with .Values.meshConfig */}}
```

Comment:
Why do we need the values above ? Do we need all of them ?

Reply:
I mean - are they 'default values' ? How many match the default values in code ( and can we fix the code to have the proper default values and do the 'if' as needed ) ?

Reply:
For backwards compatibility. I think we should leave them around for 1.6, and have it print a warning when they are used. Then in 1.7 (or really, in a few weeks when we cut the branch) we remove. What do you think?

Reply:
The defaults vs not defaults isn't really the issue here, its more about the "api" I think